### PR TITLE
docs: 0.6.7 Release notes SDK/Candid/Motoko

### PR DESCRIPTION
**Pull request by lsgunnlsgunn**
**Overview**
DRAFT release notes update
**Requirements**
What requirements are necessary to consider this problem solved?

**Considered Solutions**
What solutions were considered?

**Recommended Solution**
What solution are you recommending? Why?

**Considerations**
What impact will this change have on security, performance, users (e.g. breaking changes) or the team (e.g. maintenance costs)?

### CODE REVIEW

**Comment by chenyan-dfinity** [comment]
https://github.com/dfinity-lab/sdk/pull/991 is also user facing changes.

File: modules/release-notes/pages/sdk-release-notes.adoc [hunk 1]
```diff
@@ -35,17 +35,35 @@ This {release} version of the software and programming language should not be us
...
 +
 Because of this change to the `+dfx build+` command, the `+--skip-frontend+` option is no longer needed and has been removed.
-- There are several new `+dfx identity+` commands that enable you to create and switch between different user identities to test access control for your programs.
-- 
+* There are several new `+dfx identity+` commands that enable you to create, manage, and switch between different user identities when communicating with the {IC} network.
++
+You can also use a new `+--identity+` command-line option to set a specific user context when running `+dfx+` commands. 
++
+This new support for multiple identities enables you to test role-based access control for your programs.
+* Updates to the Candid interface description language include the following user-facing features:
```
Comment (chenyan-dfinity) [inline modules/release-notes/pages/sdk-release-notes.adoc:48]:
```suggestion
* Updates to the Candid argument parser in `dfx canister call` include the following user-facing features:
```

File: modules/release-notes/pages/sdk-release-notes.adoc [hunk 2]
```diff
@@ -35,17 +35,35 @@ This {release} version of the software and programming language should not be us
...
+....
+(42: nat8, vec {1;-3;5} : vec int8)
+....
+** Support for pretty-print decoded Candid values:
++
+`+--output pp+` is the default and prints the value in multiple lines.
++
+`+--output idl+` prints the value in a single line.
+** Support for `+float e+` notation.
+** Support for nested comments.
```
Comment (chenyan-dfinity) [inline modules/release-notes/pages/sdk-release-notes.adoc:61]:
```suggestion
```

**Comment by lsgunnlsgunn** [comment]
I took a stab at adding a couple of Motoko highlights from the changelog to the release notes. Please take a  look and let me know if there's anything I should fix, remove, or add.

**Pull request merged**